packages/SwingSet/src/kernel/state/reachable.js

```diff
@@ -14,6 +14,7 @@ export function parseReachableAndVatSlot(value) {
   } else {
     assert(`flag (${flag}) must be 'R' or '_'`);
   }
+  // @ts-expect-error xxx TS doesn't know isReachable is assigned b/c assert() throws
```

Comment:
That seems to be a legitimate mistake uncovered by tsc

```suggestion
    assert.fail(`flag (${flag}) must be 'R' or '_'`);
  }
```

Reply:
oops, yeah, I should have written `assert.fail`